tools/agentx5/cursor_cloud_agents_client.py

```
@@ -0,0 +1,64 @@
+import os
+import sys
+
+import requests
+
+BASE_URL = "https://api.cursor.com/v0"
+
+
+def auth() -> tuple[str, str]:
+    key = os.getenv("CURSOR_API_KEY", "")
+    if not key:
+        raise RuntimeError("CURSOR_API_KEY is not set")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)